composer.json

```diff
@@ -15,6 +15,12 @@
     },
     "require": {
         "php": ">=5.4",
+        "ext-pdo_mysql": "*",
+        "ext-mcrypt": "*",
+        "ext-intl": "*",
+        "ext-gd": "*",
```

Comment:
Thelia can be used without gd but with imagine. This verification is also wrong in the CheckPermission class